objectivec/ProtocolBuffers_OSX.xcodeproj/project.pbxproj

```diff
@@ -826,7 +828,7 @@
 				GCC_WARN_UNUSED_PARAMETER = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				GENERATE_PROFILING_CODE = NO;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Or is this only on the tests?  In which case the cl comment should get tweak to make it clear it isn't moving the baseline up and just for the tests because of the Swift test.